src/lib/mixer/mixer_multirotor.cpp

```diff
@@ -388,15 +388,22 @@ MultirotorMixer::mix(float *outputs, unsigned space)
 	// Slew rate limiting and saturation checking
 	for (unsigned i = 0; i < _rotor_count; i++) {
 		bool clipping_high = false;
-		bool clipping_low = false;
-
-		// check for saturation against static limits
-		if (outputs[i] > 0.99f) {
```

Comment:
Since airmode disabled only acts on the lower limit and the higher limit this saturation case should not lead to disabling the integrators. It could still be useful for log analysis to know that the mixer was in airmode thrust reduction or boosting range. Is that visible somehow?

Reply:
Yes there are more flags, like here: https://github.com/PX4/Firmware/blob/9ba4b0fdd4f4babbe78c73710fce41832ffcadbc/src/lib/mixer/mixer_multirotor.cpp#L210

Reply:
Ah, very good 👍